docs/application/dotnet/guides/nui/imageview.md

```diff
@@ -1,39 +1,101 @@
+---
+keyword: image, ImageView, subarea, premultipliedalpha, PixelArea, ResourceReady, Orientation
+---
+
 # ImageView
 
-An image view is a NUI control that displays an image. It is implemented through the `Tizen.NUI.BaseComponents.ImageView` class.
+An `ImageView` is a NUI control that displays an image. It is implemented through the [ImageView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.ImageView.html) class:
 
-**Figure: Image view example**
+## Create an ImageView
 
-![Image view example](media/ImageView.png)
+To create an image view:
 
-<a name="usage"></a>
-## Creating an Image View
+**Figure: ImageView**
 
-To create an image view:
+ ![Image View](media/ImageView.png)
 
-1.  Create an instance of the [Tizen.NUI.BaseComponents.ImageView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.ImageView.html) class and define the image file as a parameter:
+1.  To use the `ImageView` class, add the following namespaces:
 
+    ```csharp
+    using Tizen.NUI;
+    using Tizen.NUI.BaseComponents;
     ```
-    imageView = new ImageView(DirectoryInfo.Resource + "images/gallery-3.jpg");
+
+2.  It is assumed that image is in the resources directory. Create an instance of the `ImageView` class and pass the path to image file as an constructor argument:
+
+    ```csharp
+    ImageView imageView = new ImageView(DirectoryInfo.Resource + "your-image.jpg");
+    Window.Instance.Add(imageView);
     ```
 
-    You can also create the `Tizen.NUI.BaseComponents.ImageView` class instance separately and define the file path by setting its `ResourceUrl` property:
+    You can also create the `ImageView` class instance separately, and define the file path by setting its `ResourceUrl` property:
 
+    ```csharp
+    imageView = new ImageView();
+    imageView.ResourceUrl = DirectoryInfo.Resource + "your-image.jpg";
+    Window.Instance.Add(imageView);
     ```
+
+2.  To change the image path after the instance has been created, use the `SetImage()` method of the `ImageView` class:
+
+    ```csharp
     imageView = new ImageView();
-    imageView.ResourceUrl = DirectoryInfo.Resource + "images/gallery-3.jpg";
+    imageView.SetImage(DirectoryInfo.Resource + "your-image.jpg");
+    Window.Instance.Add(imageView);
     ```
+## Manipulating an Image View
```

Comment:
## Manipulate an ImageView

Reply:
Sorry I've changed figure name instead of section name before. It will be fixed in next push.